### PR TITLE
Prepare for release 1.1.14

### PR DESCRIPTION
**Pull request by gauravIoTEdge**
Prepare for Release 1.1.14

- [X] I have read the [contribution guidelines](https://github.com/azure/iotedge#contributing).

### CODE REVIEW

**Review by nlcamp — DISMISSED** [review]
LGTM

File: edgelet/version.txt [hunk 1]
```diff
@@ -1 +1 @@
-1.1.13
\ No newline at end of file
+1.1.14
```
Comment (damonbarry) [inline edgelet/version.txt]: We don't want to bump the edgelet version, since this is a Docker image-only release.

**Review by damonbarry — CHANGES_REQUESTED** [review]

File: edge-modules/metrics-collector/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+# 1.0.8 (2022-05-24)
+
+### Bug Fixes
+- Update Base Images to address Microsoft .NET Security Updates for [CVE-2022-23267](https://msrc.microsoft.com/update-guide/vulnerability/CVE-2022-23267), [CVE-2022-29117](https://github.com/dotnet/announcements/issues/220), [CVE-2022-29145](https://msrc.microsoft.com/update-guide/vulnerability/CVE-2022-29145), OpenSSL vulnerability [USN-5402-1](https://ubuntu.com/security/notices/USN-5402-1), curl vulnerability [USN-5412-1](https://ubuntu.com/security/notices/USN-5412-1), and OpenLDAP Vulnerability [USN-5424-1](https://ubuntu.com/security/notices/USN-5424-1)
```
Comment (yophilav) [inline edge-modules/metrics-collector/CHANGELOG.md:4]: Metric Collector is its own module with an independent versioning. We do it's release independently. Please remove it from here.

File: edge-modules/metrics-collector/src/config/versionInfo.json [hunk 3]
```diff
@@ -1,3 +1,3 @@
 {
-    "version": "1.0.7"
-}
\ No newline at end of file
+    "version": "1.0.8"
```
Comment (yophilav) [inline edge-modules/metrics-collector/src/config/versionInfo.json:2]: Per comment above, please undo the change.

File: edgelet/version.txt (hunk 1 above)
Comment (yophilav) [inline edgelet/version.txt:1]: We don't need to update the `edgelet` version as we do NOT update the edgelet component.

**Review by yophilav — CHANGES_REQUESTED** [review]

File: edge-modules/metrics-collector/CHANGELOG.md (hunk 2 above)
Comment (gauravIoTEdge) [reply edge-modules/metrics-collector/CHANGELOG.md:4]: We do have S360 alerts for metrics collector base images as well.

I don't mind removing it and doing the release separately -- but we would need to do a metrics collector release soon enough then.

Do you still want me to remove it?

Comment (yophilav) [reply edge-modules/metrics-collector/CHANGELOG.md:4]: Yes. Make it a separate PR.

Comment (damonbarry) [reply edge-modules/metrics-collector/CHANGELOG.md:4]: @yophilav I don't mean to throw a wrench in things...but I don't see why we can't update the release notes in this commit even though we release metrics collector separately? Both releases will point to the same commit, I don't see any problem with that.

Comment (yophilav) [reply edge-modules/metrics-collector/CHANGELOG.md:4]: Sure. I don't see harm in doing that. Personally, I'd prefer to have them as 2 separate PRs to keep my thinking straight what is being updated.

**Review by yophilav — APPROVED** [review]

File: edge-modules/metrics-collector/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,9 @@
...
+- Experimental feature to support scraping of 3rd party containers that provide prometheus metrics without iot edge specific tags. [User "alaendle" reported the issue, created the user voice feedback (that has received 12 upvotes so far), and contributed a way to enable it.]
```
Comment (micahl) [inline edge-modules/metrics-collector/CHANGELOG.md:5]:
```suggestion
- Experimental feature to [support scraping of 3rd party containers that provide prometheus metrics without iot edge specific tags](https://feedback.azure.com/d365community/idea/c5cdb9ba-398a-ec11-a81b-0022484bfd94). [Special thanks to user "alaendle" for reporting the issue, creating the user voice feedback, and contributing a potential solution.]
```

**Review by micahl — APPROVED** [review]

**Pull request merged**